pkg/kubectl/scale.go

```diff
@@ -411,27 +424,28 @@ type DeploymentScaler struct {
 }
 
 // ScaleSimple is responsible for updating a deployment's desired replicas count.
-func (scaler *DeploymentScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) error {
+func (scaler *DeploymentScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) (updatedResourceVersion string, err error) {
```

Comment:
I really disagree with this comment, actually. named return parameters are
self-documenting. No one is going to expect this to be a resource version
unless we make it a type (ideal, but too much work for this PR) or name it.

On Mon, Aug 22, 2016 at 11:22 PM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/kubectl/scale.go
> <https://github.com/kubernetes/kubernetes/pull/31051#discussion_r75808273>
> :
>
> > @@ -411,27 +424,28 @@ type DeploymentScaler struct {
> >  }
> >
> >  // ScaleSimple is responsible for updating a deployment's desired replicas count.
> > -func (scaler *DeploymentScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) error {
> > +func (scaler *DeploymentScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) (updatedResourceVersion string, err error) {
>
> nit: please make results unnamed
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/31051/files/874d5f494e22fe34ed2f48547c8da551bdc072ee#r75808273>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglhMT9Np5pTSl_ZrmYar3YL_UBbHWks5qipG3gaJpZM4Jo_lu>
> .
>